src/callbacks_step/save_solution.jl

```diff
@@ -20,13 +21,34 @@ of integration time by adding additional (shortened) time steps where necessary
 at a single point to a set of solution variables. The first parameter passed
 to `solution_variables` will be the set of conservative variables
 and the second parameter is the equation struct.
+    
+Additional nodal variables such as vorticity or the Mach number can be saved by passing a tuple of symbols
+to `extra_node_variables`, e.g., `extra_node_variables = (:vorticity, :mach)`.
+In that case the function `get_node_variables` must be defined for each symbol in the tuple.
+The expected signature of the function for (purely) hyperbolic equations is:
+```julia
+function get_node_variables(::Val{symbol}, u, mesh, equations, dg, cache)
+    # Implementation goes here
+end
+```
+while for parabolic-hyperbolic equations `equations_parabolic` and `cache_parabolic` must be added:
+```julia
+function get_node_variables(::Val{symbol}, u, mesh, equations, dg, cache,
+                             equations_parabolic, cache_parabolic)
+    # Implementation goes here
+end
+```
+
+In case that the [`SubcellLimiterIDP`](@ref) is used, the `extra_node_variables` tuple is automatically extended by the 
+`:limiting_coefficient` key which contains the limiting coefficient for each node.
```

Comment:
TODO: Change the `if`-clauses in the `function get_node_variables!` to really check for IDP and not only subcell vol integral (see also failing downstream tests)